src/ctest.ts

```diff
@@ -117,65 +117,64 @@ function decodeOutputMeasurement(node: EncodedMeasurementValue | string): string
         // XML parsing is obnoxious. This condition means that there are no tests,
         // but CTest is still enabled.
         return {
-            Site: {
-                $: messy.Site.$,
-                Testing: {
-                    TestList: [],
-                    Test: []
+            site: {
+                $: messy.site.$,
+                testing: {
+                    testList: [],
+                    test: []
                 }
             }
         };
     }
     return {
-        Site: {
-            $: messy.Site.$,
-            Testing: {
-                TestList: testing_head.TestList.map(l => l.Test[0]),
-                Test: testing_head.Test.map((test): Test => ({
-                    FullName: test.FullName[0],
-                    FullCommandLine: test.FullCommandLine[0],
-                    Name: test.Name[0],
-                    Path: test.Path[0],
-                    Status: test.$.Status,
-                    Measurements: new Map<string, TestMeasurement>(),
-                    Output: decodeOutputMeasurement(test.Results[0].Measurement[0].Value[0])
+        site: {
+            $: messy.site.$,
+            testing: {
+                testList: testingHead.testList.map(l => l.test[0]),
+                test: testingHead.test.map((test): Test => ({
+                    fullName: test.fullName[0],
+                    fullCommandLine: test.fullCommandLine[0],
+                    name: test.name[0],
+                    path: test.path[0],
+                    status: test.$.status,
+                    measurements: new Map<string, TestMeasurement>(),
+                    output: decodeOutputMeasurement(test.results[0].measurement[0].value[0])
                 }))
             }
         }
     };
 }
 
-export async function readTestResultsFile(test_xml: string) {
-    const content = (await fs.readFile(test_xml)).toString();
+export async function readTestResultsFile(testXML: string) {
+    const content = (await fs.readFile(testXML)).toString();
     const data = await parseXMLString(content) as MessyResults;
     const clean = cleanupResultsXML(data);
     return clean;
 }
 
 export function parseCatchTestOutput(output: string): FailingTestDecoration[] {
-    const lines_with_ws = output.split('\n');
-    const lines = lines_with_ws.map(l => l.trim());
+    const wsLines = output.split('\n');
+    const lines = wsLines.map(l => l.trim());
     const decorations: FailingTestDecoration[] = [];
     for (let cursor = 0; cursor < lines.length; ++cursor) {
         const line = lines[cursor];
         const regex = process.platform === 'win32' ? /^(.*)\((\d+)\): FAILED:/ : /^(.*):(\d+): FAILED:/;
-        const res = regex.exec(line);
-        if (res) {
-            const [_all, file, lineno_] = res;
-            void _all;  // unused
-            const lineno = parseInt(lineno_) - 1;
+        const result = regex.exec(line);
+        if (result) {
+            const [, file, lineNumber] = result;
```

Comment:
call this one 'arg2' and use 'lineNumber' for the actual integer (replace 'lineno' below).